lib/cocoapods/command.rb

```diff
@@ -121,5 +122,25 @@ def verify_lockfile_exists!
         raise Informative, "No `Podfile.lock' found in the project directory, run `pod install'."
       end
     end
+
+    def self.verify_xcode_license_approved!
+      unless xcode_license_approved?
+        UI.puts xcode_license_message.red
```

Comment:
i'm not sure about this, it makes for uglier output in my opinion.

### neat
![screen shot 2015-02-13 at 10 00 36 am](https://cloud.githubusercontent.com/assets/300235/6189381/22429b1c-b368-11e4-8ee5-33399724bb5f.png)


### ugly
![screen shot 2015-02-13 at 10 05 38 am](https://cloud.githubusercontent.com/assets/300235/6189354/f268d69a-b367-11e4-9816-9f990e078d8b.png)